moped-etl/arcgis/utils.py

```diff
@@ -0,0 +1,154 @@
+import os
+
+import requests
+
+from settings import SERVICE_NAMES
+
+AGOL_USERNAME = os.getenv("AGOL_USERNAME")
+AGOL_PASSWORD = os.getenv("AGOL_PASSWORD")
+HASURA_ENDPOINT = os.getenv("HASURA_ENDPOINT")
+HASURA_ADMIN_SECRET = os.getenv("HASURA_ADMIN_SECRET")
+AGOL_ORG_BASE_URL = "https://austin.maps.arcgis.com"
+
+
+def get_endpoint(method, feature_type):
```

Comment:
i opted to write out the AGOL operations directly against the REST API, instead of using Esri's `arcgis` [library](https://developers.arcgis.com/python/). That library is very, very large and is total overkill for the add/update tasks at hand. It also didn't install cleanly for me on my ARM chip (👍👍👍👍 ).

so this is a slight break from [knack services](https://github.com/cityofaustin/atd-knack-services/blob/production/services/records_to_agol.py), which does use the `arcgis` library to do similar work. and more of a throwback to the old [agolutil](https://github.com/cityofaustin/atd-utils-agol/blob/master/agolutil/agolutil.py).

Reply:
sounds great to me and agree that ArcGIS endpoints are 🤯! Airflow will also be happy to tell us when/if anything breaks once this is running routinely. It doesn't sound too risky to me. Edit: The code below for the requests is straightforward to read too.